tests/integration/components/volunteer-headshot-test.js

```diff
@@ -0,0 +1,73 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import volunteerHeadshot from '../../pages/components/volunteer-headshot';
+
+let page = PageObject.create(volunteerHeadshot);
+
+const roles = [
+  'Developer',
+  'Ember Developer',
+  'UX Designer',
+  'Software Engineer',
+  'Project Coordinator',
+  'Designer & Developer'
+];
+
+function generateUserRoles() {
```

Comment:
Since we are just adding 1 object per role to this array (i.e. the number of roles never varies), thoughts on just making it static to start with:

```
const usersRoles = [
  { role: { name: 'Developer' } },
  { role: { name: 'Ember Developer' } },
  { role: { name: 'UX Designer' } },
  { role: { name: 'Software Engineer' } },
  { role: { name: 'Project Coordinator' } },
  { role: { name: 'Designer & Developer } }'
];
```
This way we aren't rebuilding this on every test.